### PR TITLE
Add Persist Redis module as a Ballerina Extended module

### PR DESCRIPTION
**Pull request by dinukaamarasinghe817**
## Purpose
To list `persist.redis` as a Ballerina Extended module in Ballerina Library

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,12 +80,13 @@ Ballerina code is distributed under [Apache license 2.0](https://github.com/ball
...
...m/module-ballerinax-persist.redis/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-persist.redis)        |        [![Bugs](https://img.shields.io/github/issues-search/ballerina-platform/ballerina-library?query=is%3Aopen%20label%3Amodule%2Fpersist.redis%20label%3AType%2FBug&label=&color=30c955)](https://github.com/ballerina-platform/ballerina-library/issues?q=is%3Aopen%20label%3Amodule%2Fpersist.redis%20label%3AType%2FBug)        |        [![GitHub Pull Requests](https://img.shields.io/github/issues-pr-raw/ballerina-platform/module-ballerinax-persist.redis.svg?label=)](https://github.com/ballerina-platform/module-ballerinax-persist.redis/pulls)        |    [![Load Tests](https://img.shields.io/badge/-N%2FA-yellow)](https://github.com/ballerina-platform/module-ballerinax-persist.redis/actions/workflows/process-load-test-result.yml)     |        [![GraalVM Check](https://img.shields.io/github/actions/workflow/status/ballerina-platform/module-ballerinax-persist.redis/build-with-bal-test-graalvm.yml?branch=main&label=)](https://github.com/ballerina-platform/module-ballerinax-persist.redis/actions/workflows/build-with-bal-test-graalvm.yml)        |
```
Comment (daneshk) [inline README.md:90]: we don't need to update the readme file. it will update automatically

Comment (dinukaamarasinghe817) [reply README.md:89]: Fixed in https://github.com/ballerina-platform/ballerina-library/pull/6165/commits/ff2c48b1d541f7a457a6d9c7aebbdef8e93ba673

Comment (daneshk) [reply README.md:89]: Let's remove all changes in README.md

Comment (dinukaamarasinghe817) [reply README.md]: Fixed in https://github.com/ballerina-platform/ballerina-library/pull/6165/commits/62cb6c30798d2a0ce18522f6a8a8eba00ee1d80c

**Review by daneshk — APPROVED** [review]

**Pull request merged**